### PR TITLE
docs(react-native): clarify `getContentCards()` vs. `requestContentCardsRefresh()`

### PR DESCRIPTION
**Pull request by jrmykolyn**
# Pull Request/Issue Resolution

#### Description of Change:
This pull request updates the [React Native content card integration guide](https://www.braze.com/docs/developer_guide/platform_integration_guides/react_native/content_cards/#customization). 

Specifically, it enhances the documentation for the `getContentCards()` and `requestContentCardsRefresh()` SDK methods. This as been an area of confusion for myself and my team, and I believe describing their behaviour in more detail will prevent confusion among _future_ Braze React Native SDK consumers.

There is no corresponding issue for this pull request.

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [X] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [ ] Check that you haven't removed any images (replacing an image with an updated one of the same name is fine), as this breaks the French site
- [ ] Check that all links work.
- [ ] Ensure you have completed [our Contributors License Agreement](https://www.braze.com/docs/cla/).
- [ ] Tag @josh-mccrowell-braze and @internetisaiah as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the internal reviewing chart to tag the appropriate reviewer.
- [ ] If the documentation involves a 1) paid SKU, 2) a third party, 3) SMS, 4) AI, or 5) privacy, ensure that Ana Teresa Serafino on the Legal team has signed off.
- [ ] Tag others as reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub)
- [Image Style Guide](https://docs.google.com/document/d/e/2PACX-1vRJSkwcjmjrTfLDagZccLpOMMyh5NN5SXRZSjz12cRAHbX4OrUmhvCmYpf_p5YB-9r4_jSOQLkicQIH/pub)
- [Styling Test Page](https://www.braze.com/docs/home/styling_test_page/)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Vercel environment by clicking the preview link in the vercel-bot comment in your PR.
</details>

<details>
<summary>❗ ATTN: Internal Reviewing Chart </summary>
<br>
<b>Work at Braze and not sure who to tag for review?</b> <br>Before tagging @josh-mccrowell-braze or @internetisaiah for a general review, reference the following chart to see if a specific product vertical/reviewer applies to your pull request.
<br><br>
<table>
<tr>
    <td><b>Reviewer</b></td>
    <td><b>Product Vertical</b></td>
  </tr>
  <tr>
    <td>@josh-mccrowell-braze</td>
    <td>Monolith Deployments<br>Quality Infrastructure<br>Platform Infrastructure<br>Datalake<br>SDKs<br>Currents</td>
  </tr>
  <tr>
    <td>@internetisaiah</td>
    <td>Developer Guide<br>Technology partners<br>Channels</td>
  </tr>
  <tr>
    <td>@bre-fitzgerald</td>
    <td>Intelligence<br>In-App Messages<br>Channels<br>Frontend Infrastructure & Experience (FIX)</td>
  </tr>
  <tr>
    <td>@lydia-xie</td>
    <td>Ingestion<br>Core Objects<br>Core Messaging<br>Messaging Experience<br>Message Components<br>Email (Composition and Infrastructure) (tag @rachel-feinberg for Liquid use cases)</td>
  </tr>
  <tr>
    <td>@rachel-feinberg</td>
    <td>Customer Lifecycle, Identity and Permissions<br>SMS<br>User Targeting<br>Reporting</td>
  </tr>
</table>
</details>

### CODE REVIEW

**Comment by jrmykolyn** [comment]
Hi @internetisaiah,

The checklist mentions that I should tag you when the pull request is ready for review.

 I see that there are some failing CI/CD integrations, which I suspect are related to missing translations (ie. into French and Japanese). Given that I'm not fluent in _either_, do you have any guidance on next steps?
 
 Thanks a lot.

**Comment by internetisaiah** [comment]
hi @jrmykolyn , nice to e-meet you! your deployments have been authorized 👍🏽 to avoid our deploy bots from being overloaded by outside contributors (or forks), they have to be manually approved before building. just let me know when this one's ready for review!

**Comment by jrmykolyn** [comment]
@internetisaiah Great, thanks for the prompt response.

The update itself is a minor copy change to the React Native SDK documentation. It's currently English only, and I have no reliable way to translate the update into French and Japanese.

Is that a blocker for review?

**Comment by internetisaiah** [comment]
@jrmykolyn , nope! good question. our translation team handles all translations in bulk. typically docs pages get deployed to `master` in english, then are translated internally once they go live. so nothing to do on your end!

**Comment by jrmykolyn** [comment]
@internetisaiah Great, then let's call this one _ready for review_.

**Comment by jrmykolyn** [comment]
@internetisaiah Is there anything I can do to help get this reviewed? Thanks!

**Comment by internetisaiah** [comment]
hey @jrmykolyn , thanks for checking in again! due to other reviews i'm working on, this probably won't be ready until the next tuesday deployment on July 2nd. if you and your team need this done sooner, just let me know and i can see if there's someone else on my team that has bandwidth to pick this up. thanks for understanding!

**Comment by josh-mccrowell-braze** [comment]
hey @internetisaiah and @jrmykolyn - this is a small lift, I'll go ahead and grab this one!

File: _docs/_developer_guide/platform_integration_guides/react_native/content_cards.md [hunk 1]
```diff
@@ -42,8 +42,8 @@ You can use these additional methods to build a custom Content Cards Feed within
 | Method                                   | Description                                                                                            |
 | ---------------------------------------- | ------------------------------------------------------------------------------------------------------ |
 | `launchContentCards()`                   | Launches the Content Cards UI element.                                                                 |
-| `requestContentCardsRefresh()`           | Requests the latest Content Cards from the Braze SDK server.                                           |
-| `getContentCards()`                      | Retrieves Content Cards from the Braze SDK. This will return the latest list of cards from the server. |
+| `requestContentCardsRefresh()`           | Requests the latest Content Cards from the Braze SDK server. The resulting list of cards is passed to each of the _previously_ registered [content card event listeners](#customization). |
```
Comment (josh-mccrowell-braze) [inline _docs/_developer_guide/platform_integration_guides/react_native/content_cards.md:45]:
```suggestion
| `requestContentCardsRefresh()`           | Requests the latest Content Cards from the Braze SDK server. The resulting list of cards is passed to each of the previously registered [content card event listeners](#customization). |
```

File: _docs/_developer_guide/platform_integration_guides/react_native/content_cards.md [hunk 2]
```diff
@@ -42,8 +42,8 @@ You can use these additional methods to build a custom Content Cards Feed within
...
+| `getContentCards()`                      | Retrieves Content Cards from the Braze SDK. This returns a Promise that resolves with the latest list of cards from the server. |
```
Comment (josh-mccrowell-braze) [inline _docs/_developer_guide/platform_integration_guides/react_native/content_cards.md:46]:
```suggestion
| `getContentCards()`                      | Retrieves Content Cards from the Braze SDK. This returns a promise that resolves with the latest list of cards from the server. |
```

**Review by josh-mccrowell-braze — APPROVED** [review]

**Comment by josh-mccrowell-braze** [comment]
@cla-bot check

**Pull request merged**